internal/pkg/manifest/svc.go

```diff
@@ -41,6 +42,72 @@ type ServiceImageWithPort struct {
 	Port         uint16 `yaml:"port"`
 }
 
+// LogConfig holds configuration for Firelens to route your logs.
+type LogConfig struct {
+	Destination    destinationConfig `yaml:"destination,flow"`
+	EnableMetadata *bool             `yaml:"enableMetadata"`
```

Comment:
Out of curiosity is the default for us "false"? should it be true instead

Reply:
i think it is default to be true. But since the whole LogConfig is hidden so we don't make it enabled in our manifest. However, if a user configures `logging` then we'll default `EnableMetadata` to be true when deploying the svc.

Reply:
Perfect 👌